### ISSUE TITLE
QQ, WeChat and Weibo Support

### ISSUE THREAD

**Issue by Cheedoong** [lipp/login-with#46]
May you consider supporting the largest platforms in China?

Reference: 
https://github.com/anerg2046/sns_auth

**Comment by lipp** [comment]
Of course! There seems to be passport modules for [qq](https://www.npmjs.com/package/passport-qq), [wechat](https://www.npmjs.com/package/passport-wechat) and [weibo](https://www.npmjs.com/package/passport-weibo). Adding a new "platform/strategy" when you have a passport module is pretty simple (have a look at #45 ).

PRs are very welcomed!

Thanks for the important feedback.

**Comment by lipp** [comment]
Would be awesome to get the help from someone who can speak chinese... I have some problems creating an account on www.qq.com :(

**Comment by Cheedoong** [comment]
You may also extract code from: 
https://github.com/Yurunsoft/YurunOAuthLogin

I think we (Pear Limited) can assign an engineer to help you. 
@airchen233 would like to help?

**Comment by airchen233** [comment]
Glad to help. And we are good at speaking chinese~

**Comment by lipp** [comment]
😃  👍  Awesome!
